exe/facter.cc

```diff
@@ -299,6 +281,9 @@ int main(int argc, char **argv)
             log(level::warning, "Facter was called recursively, skipping external facts. Add '--no-external-facts' to silence this warning");
           } else {
             environment::set("INSIDE_FACTER", "true");
+            if (vm.count("external-dir")) {
+                external_directories.push_back(vm["external-dir"].as<string>());
```

Comment:
vm["external-dir"] is listed as supporting a vector of strings. How does this work?